tests/fixtures/ead/ead_record_all_fields.xml

```diff
@@ -1,32 +1 @@
-            <datestamp>2021-11-06T14:35:55Z</datestamp>
-        </header>
-        <metadata>
-            <ead xmlns="urn:isbn:1-931666-22-9" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:isbn:1-931666-22-9 http://www.loc.gov/ead/ead.xsd">
-                <eadheader countryencoding="iso3166-1" dateencoding="iso8601" findaidstatus="completed" langencoding="iso639-2b" repositoryencoding="iso15511">
-                    <eadid countrycode="US" mainagencycode="US-mcm">VC-0002</eadid>
-                    <filedesc>
-                        <titlestmt>
-                            <titleproper>
-                                Guide to the Charles J. Connick Stained Glass Foundation Collection
-                                <num>VC.0002</num>
-                            </titleproper>
-                            <titleproper>
-                                Title 2
-                                <num>VC.0002</num>
-                            </titleproper>
-                            <titleproper>
-                                Title 3
-                            </titleproper>
-                            <titleproper></titleproper>
-                        </titlestmt>
-                    </filedesc>
-                </eadheader>
-            </ead>
-        </metadata>
-    </record>
-</records>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?> <records> <record xmlns="http://www.openarchives.org/OAI/2.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"> <header> <identifier>oai:mit//repositories/2/resources/1</identifier> <datestamp>2021-11-06T14:35:55Z</datestamp> </header> <metadata> <ead xmlns="urn:isbn:1-931666-22-9" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:isbn:1-931666-22-9 http://www.loc.gov/ead/ead.xsd"> <eadheader countryencoding="iso3166-1" dateencoding="iso8601" findaidstatus="completed" langencoding="iso639-2b" repositoryencoding="iso15511"> <eadid countrycode="US" mainagencycode="US-mcm">VC-0002</eadid> </eadheader> <archdesc level="collection"> <did> <unittitle>Charles J. Connick Stained Glass <emph>Foundation <emph>Collection</emph></emph><num>VC.0002</num> </unittitle> <unittitle> Title 2 <num>VC.0002</num> </unittitle> <unittitle>Title 3</unittitle> <origination label="Creator"> <persname authfilenumber="nr99025157" rules="rda" source="naf">Connick, Charles J. <part>(<emph>Charles <emph>Jay</emph></emph>)</part></persname> <persname authfilenumber="nr9902" rules="rda">Author, Unknown</persname> <persname authfilenumber="nr97" source="viaf">Name 2</persname><famname authfilenumber="nr9957" source="snac">Name <part>3</part></famname> <famname>Name 4</famname> </origination> <origination> <persname authfilenumber="nr99025435" source="viaf">Name 5</persname> </origination> </did> </archdesc> </ead> </metadata> </record> </records>
```

Comment:
Agree on readability but I was finding as soon as I do that, values get ugly with carriage returns in any value with nested XML:

```
Contributor(value='Connick, Charles J. (Charles Jay)', affiliation=None, identifier=['https://lccn.loc.gov/nr99025157'], kind='Creator', mit_affiliated=None)
```

becomes 

```
Contributor(value='\n                                Connick, Charles J.\n                                \n                                    (\n                                    \n                                        Charles\n                                        Jay\n\n                                    )\n                                \n', affiliation=None, identifier=['https://lccn.loc.gov/nr99025157'], kind='Creator', mit_affiliated=None)
```
Is there a workaround?

Reply:
Not entirely sure yet, but it's something we're going to have to deal with because the harvested records we're getting do contain both newlines and lots of extra whitespace in the mixed value fields.

Reply:
I'll play around with the indented fixture, some `if` checks and `strip()` calls may take care of it

Reply:
I think the best bet is to change `parse_mixed_value()` to:
```
if type(item) == NavigableString and item:
    yield str(item.strip())
...
```
Doing this will mean we probably want to change the default separator value for `create_string_from_mixed_value()` to `" "`, and some of the result values may end up slightly weird (e.g. `'Connick, Charles J. ( Charles Jay )'` instead of `'Connick, Charles J. (Charles Jay)'`, but that seems far preferable to me than including all the extraneous whitespace from the source records...

Also remembered while looking at this that anywhere we use a `NavigableString` in the timdex record, we need to convert it to a unicode string first (as above) per the bs4 docs.

Reply:
Agree that's the lesser of two evils

Reply:
Hmm, I don't think that should be necessary. If the `parse_mixed_value` recursion is working properly it should only ever return stripped strings. Did you update the rest of it based on my comment on the for loop?

Reply:
Huh, I'm seeing the same thing in the contributor value (as in my comment above) but not for title...

Reply:
Oh. Just realized I had locally changed the if check in `parse_mixed_value` to `if type(item) == NavigableString and item.strip()` because that way we never even bother with values that are only whitespace.

Reply:
Oh, that did it, thanks!